python/paddle/fluid/tests/unittests/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ if(APPLE)
     # TODO: add the unitest back when it fixed
     list(REMOVE_ITEM TEST_OPS test_detection_map_op)
     list(REMOVE_ITEM TEST_OPS test_dist_se_resnext)
+    # TODO(tangwei12): add the unitest back when it fixed
+    list(REMOVE_ITEM TEST_OPS test_dist_word2vec)
```

Comment:
Maybe remove it only on Mac?

Reply:
the UT is necessary on other platforms, I will fix it on Mac in private.